server/policies.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  impersonate: async (req, res, {id}) => {
+    if (!id || !req.tokenPayload) throw new Error('403 Forbidden')
```

Comment:
This should throw an integer as sails-async handles `Errors` differently. This isn't an error